ci-operator/step-registry/hypershift/kubevirt/create/hypershift-kubevirt-create-chain.yaml

```diff
@@ -7,15 +7,18 @@ chain:
     - name: HYPERSHIFT_NODE_COUNT
       default: "3"
       documentation: "The number nodes to automatically create and join to the cluster."
+    - name: HYPERSHIFT_CLI
+      default: "hypershfit"
+      documentation: "The chosen CLI, hcp or hypershift"
     commands: |-
       set -exuo pipefail
 
       HCP_CLI="/usr/bin/hcp"
-      if [[ ! -f $HCP_CLI ]]; then
+      if [[ ! -f $HCP_CLI && "${HYPERSHIFT_CLI}" == "hcp" ]]; then
           # we have to fall back to hypershift in cases where the new hcp cli isn't available yet
           HCP_CLI="/usr/bin/hypershift"
       fi
-      echo "Using $HCP_CLI for cli"
+      echo "Using $HYPERSHIFT_CLI for cli"
```

Comment:
the `HYPERSHIFT_CLI` env var isn't actually used when invoking the cli. It's the `HCP_CLI` env var that's used when performing the create command